config/config.sample.edn

```diff
@@ -9,6 +9,7 @@
   :google {:api {:key ""}
            :custom {:search {:engine {:id ""}}}
            :options {:safe "high"}}
+  :discord {:token ""}
```

Comment:
This one's unused, right?

Reply:
Oh yeah you're probably right. Sorry, I was just trying to figure out how everything worked together haha. Will remove